CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contributing to Stryker Dashboard
+
+This is the contribution guide for Stryker Dashboard. Great to have you here! Here are a few ways you can help make this project better.
+
+## Creating issues
+
+Do you have an idea for a feature or have you found a bug? Please create an issue so we can talk about it!
+
+## Adding new features
+
+New features are welcome! Either as requests or proposals.
+
+1. Please create an issue first, so we know what to expect from you.
+2. Create a fork on your github account.
+4. Please create or edit unit tests or integration tests.
+5. Run the tests and verify they pass.
+6. Update documentation.
+
+## Maintainers
+
+### Using Milestones
+
+Milestones are used to group issues and pull requests together to provide a clear overview of the project's progress. When submitting or reviewing issues, please consider the following guidelines related to milestones:
+
+1. Assigning Milestones: When creating or reviewing an issue, consider whether it aligns with any existing milestones. If it does, please assign the appropriate milestone to the issue to ensure it is properly tracked.
+
+2. Creating New Milestones: If you believe that a new milestone is necessary to better organize and track the project's progress, please consult with the project maintainers before creating a new milestone.
+
+3. Tracking Progress: By utilizing milestones, we can collectively track progress and better understand the project's development roadmap.
+
+4. Prioritizing Issues: Milestones can also help in prioritizing issues and feature requests, providing a clearer picture of what needs to be addressed in each development cycle.
+
+By adhering to these guidelines, we can ensure that the use of milestones enhances the clarity and organization of our project's development process. Once a milestone is complete, we can release the next version.
+
+### Naming conventions
```

Comment:
In my opinion this convention isn't something contributors should have to think about. They should be able to name their PR and commits anything they want, and before squash-merging, one of the maintainers can change it to the correct convention for changelogs